src/libraries/System.Private.CoreLib/src/System/Resources/ManifestBasedResourceGroveler.cs

```diff
@@ -231,45 +231,16 @@ internal ResourceSet CreateResourceSet(Stream store, Assembly assembly)
                     }
                     else
                     {
-                        IResourceReader reader;
-
-                        // Permit deserialization as long as the default ResourceReader is used
-                        if (ResourceManager.IsDefaultType(readerTypeName, ResourceManager.ResReaderTypeName))
```

Comment:
Did we remove this check? `ResourceManager.IsDefaultType`?

Reply:
Oh, nevermind. We are creating a ResourceSet through reflection later even in this case.